test/test.go

```diff
@@ -28,6 +28,80 @@ func testEmbed(g abstract.Group, rand cipher.Stream, points *[]abstract.Point,
 	*points = append(*points, p)
 }
 
+func testPointSet(g abstract.Group, rand cipher.Stream) {
+	N := 1000
+	null := g.Point().Null()
+	for i := 0; i < N; i++ {
```

Comment:
Ok you agreed but that does not say why you agreed to keep it ? What is the benefit of doing it a thousand time ?
There's no concurrency here, no external actor, no statistics involved and so on ...